### PR TITLE
Update import-hr-data.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that some customers may have to whitelist the URL https://webhook.ingestion.office.com

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11027

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/import-hr-data.md [hunk 1]
```diff
@@ -312,6 +312,9 @@ If you haven't already done so, you can copy the values for the **Azure App ID**
 
 You can also select **Edit** to change the Azure App ID or the column header names that you defined on the **File mapping** page.
 
+> [!NOTE]
+> If you have trouble running the HR connector script, you can whitelist the URL 'https://webhook.ingestion.office.com' and try again.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/import-hr-data.md]:
```suggestion
> If you have trouble running the HR connector script, you can add the URL 'https://webhook.ingestion.office.com' to the allowlist and try again.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @msbemba, just for reference, we never use "whitelist" in Microsoft docs. Thanks.

**Comment by yogkumgit** [comment]
@msbemba gentle reminder

**Comment by yogkumgit** [comment]
@msbemba Please update the doc with feedback. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR.Thanks

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder

**Comment by yogkumgit** [comment]
Closing as per the comment in #11027

**Pull request closed without merging**